lib/util.js

```diff
@@ -187,9 +187,13 @@ const util = {
 
     if (process.platform === 'win32') util.updateOmahaMidlFiles()
 
+    let num_compile_failure = 1
+    if (config.ignore_compile_failure)
+      num_compile_failure = 1000
```

Comment:
maybe we want to allow more than 1k? I'm thinking chromium rebase sometimes might surpass since 1 real error can be much more. Merging in any case though.